cmd/ui/src/views/GroupManagement/GroupManagement.tsx

```diff
@@ -72,7 +70,8 @@ const GroupManagement = () => {
             };
         });
     };
-
+    console.log('globalDomain', globalDomain);
+    if (!globalDomain) return null;
```

Comment:
`globalDomain` is initiated as null so it doesnt hurt to have this check. Also TS wants it to be able to pass to globalDomain as a prop to `GroupManagementContent`.